pkg/serverless/trace/propagation/carriers.go

```
@@ -220,3 +221,55 @@ func rawPayloadCarrier(rawPayload []byte) (tracer.TextMapReader, error) {
 func headersCarrier(hdrs map[string]string) (tracer.TextMapReader, error) {
 	return tracer.TextMapCarrier(hdrs), nil
 }
+
+// createTraceContextFromStepFunctionInput extracts the execution ARN, state name, and state entered time and uses them to generate Trace ID and Parent ID
+func createTraceContextFromStepFunctionInput(event events.StepFunctionEvent) (*TraceContext, error) {
+	tc := new(TraceContext)
+
+	execArn := event.Execution.ID
+	stateName := event.State.Name
+	stateEnteredTime := event.State.EnteredTime
```

N.B. Maybe I should return an error if these are empty?